examples/plot_casino_example.py

```diff
@@ -87,24 +87,22 @@
 
 best_score = best_model = None
 n_fits = 50
-np.random.seed(13)
+rng = np.random.default_rng(13)
```

Comment:
We don't require a version of numpy that necessarily has the new RNG API, and I would rather not bump the requirement just for that, so let's stick to the old RNG API.